Yoast/wordpress-seo
# Subscribe to Gutenberg's data API

## Summary

This PR can be summarized in the following changelog entry: 

* Makes it possible to use the content, slug, title and excerpt from Gutenberg posts in the SEO and readability analysis.

## Relevant technical choices:

* This is a prototype. Architecture and naming might change in the future.

## Test instructions

This PR can be tested by following these steps:

* Create a post in Gutenberg.
* Make sure all SEO and readability assessments that use the content, slug and/or title work correctly.
* Also make sure all assessments still work correctly in the classic editor.
* Write an excerpt in the document sidebar. Use the %%excerpt%% and %%title%% replace var in the snippet preview.

**Important**: Currently, Gutenberg only saves a slug after publishing the post.

## Quality assurance


Fixes #9019
